### PR TITLE
add guide on deploying models using seldon

### LINKED ISSUE #46: Add documentation on Model deployment steps
We are pretty comfortable on working with Seldon to deploy our machine learning models. We should add some documentation on how to deploy models on OpenShift to our ds workflows and a section to the [Get Started page](https://www.operate-first.cloud/data-science/data-science-workflows/docs/getting-started.md) on model deployment.

This [document](https://docs.google.com/document/d/1lFTcHhawvVYPSY2lnLpImSeVHQp7fD_yKhtYM8tobD4/edit?usp=sharing) can be a good starting point.

### LINKED ISSUE #61: Add how to deploy models using seldon section to Getting Started
- [ ] Convert [document](https://docs.google.com/document/d/1lFTcHhawvVYPSY2lnLpImSeVHQp7fD_yKhtYM8tobD4/edit#heading=h.of1xdr8qwgdl) into a guide and add a section in the [Getting Started](https://github.com/aicoe-aiops/data-science-workflows/blob/master/docs/getting-started.md) document

### PR DESCRIPTION
**Pull request by oindrillac**
resolves #61 
resolves #46

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -79,6 +79,9 @@ Here are potential ways to contribute and showcase work you have completed.
 ### **Creating Jupyterhub Images**
 JH images can be made from your projects to easily share reproducible ML experiments and create interactive Data Science working environments. Please refer to the following [guide](https://www.operate-first.cloud/users/data-science-workflows/docs/create_and_deploy_jh_image.md) for more information.
 
+### Deploy models on OpenShift using Seldom.
```
Comment (aakankshaduggal) [inline docs/getting-started.md:82]: nit :
 seldom -> Seldon

Comment (oindrillac) [reply docs/getting-started.md]: fixed :)

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm

File: docs/deploy-models-using-seldon.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Deploy models on OpenShift using Seldon Server
+
+Have a model trained and ready to share with the world? In this guide, we go over how to deploy models as a microservice using one of Seldon Core’s pre-packaged [model servers](https://docs.seldon.io/projects/seldon-core/en/v1.1.0/servers/overview.html).
+
+## Pre-requisites
+
+* An ML model trained using one of these supported ML [libraries](https://docs.seldon.io/projects/seldon-core/en/latest/nav/config/servers.html) or a custom model (guide forthcoming) .
+* OpenShift namespace with [seldon operator installed](https://docs.seldon.io/projects/seldon-core/en/latest/workflow/install.html#openshift) and administrator privileges available.
```
Comment (MichaelClifford) [inline docs/deploy-models-using-seldon.md:8]: Is there anything we can add here about, using existing OPF resources? like "(see below for instructions on accessing the opf seldon instances)" or something?

File: docs/deploy-models-using-seldon.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    metadata:
+    name: seldon-init-container-secret
+    type: Opaque
+    data:
+    AWS_ACCESS_KEY_ID: XXXX
+    AWS_SECRET_ACCESS_KEY: XXXX
+    AWS_ENDPOINT_URL: XXXX
+    USE_SSL: XXXX
+    ```
+    Find documentation on how to configure different object storage providers [here](https://docs.seldon.io/projects/seldon-core/en/latest/servers/kfserving-storage-initializer.html).
```
Comment (MichaelClifford) [inline docs/deploy-models-using-seldon.md:37]: should this be italicized? feels like a note referencing the above, but given the block of code and following instruction seems confusing as to what is is referring to.

File: docs/deploy-models-using-seldon.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    
+    ![create deployment](../public/assets/deploy-models-using-seldon/create-deployment.png)
+
+    * If you are using the SKLearn Seldon server, the `modeluri` refers to the model’s prefix in the Ceph bucket. In your Seldon Deployment, change model URI to be for your model. Sample model URI from S3: `s3://<bucket_name>/<project_name>/<model_name>/model.joblib`
+    
+    * In your Seldon Deployment, add a reference to the secret with the AWS credentials for access to the bucket like `envSecretRefName: seldon-init-container-secret`, where seldon-init-container-secret is the secret name.
+  
+    * To test the deployed model endpoint, create a route to the deployed model by selecting the deployed service. To create a route, from the OpenShift console, go to `Networking`, then go to `Routes` and click on “Create Route”. Give the route a desired name, and select the service that we deployed earlier, select a “Target Port” and click on Create.
+
+    ![create route](../public/assets/deploy-models-using-seldon/create-route.png)
```
Comment (MichaelClifford) [inline docs/deploy-models-using-seldon.md:52]: can we add a short "test it out" section here with an example curl command or something?

**Review by MichaelClifford — CHANGES_REQUESTED** [review]
Nice and clear tutorial :+1: just a couple of comments.

File: docs/deploy-models-using-seldon.md (hunk 2 above)
Comment (oindrillac) [reply docs/deploy-models-using-seldon.md:8]: let me see if we have any existing Seldon related documentation on opf

Comment (oindrillac) [reply docs/deploy-models-using-seldon.md:8]: Could not find any existing access docs on seldon. Issue -https://github.com/operate-first/apps/issues/1413

Comment (MichaelClifford) [reply docs/deploy-models-using-seldon.md:8]:
```suggestion
* OpenShift namespace with [seldon operator installed](https://docs.seldon.io/projects/seldon-core/en/latest/workflow/install.html#openshift) and administrator privileges available. (If you are using the [Operate First cluster](https://console-openshift-console.apps.zero.massopen.cloud/), you can use the namespace `ds-ml-workflows-ws` which already has Seldon deployed) 
```

**Review by MichaelClifford — CHANGES_REQUESTED** [review]
small edit :)

File: docs/deploy-models-using-seldon.md (hunk 2 above)
Comment (oindrillac) [reply docs/deploy-models-using-seldon.md]: changes made :)

**Review by MichaelClifford — APPROVED** [review]

**Comment by MichaelClifford** [comment]
Prow is down atm, this is only a text change. Merging.

**Pull request merged**